genrl/deep/bandit/dcb_agents/common.py

```diff
@@ -0,0 +1,406 @@
+import random
+from typing import List, Optional, Tuple, Union
+
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+from torch import dropout, dropout_
+from torch.cuda import init
+
+device = torch.device("cuda" if torch.cuda.is_available() else "cpu")
+dtype = torch.float
```

Comment:
Same comment about the necessity of these variables? My issue really is that, if even one of these files is imported somewhere, these variables are implicitly imported as well